src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -51,7 +58,10 @@ static const TypeMapping dnnlFCTypeMapping {
     // quantization configuration (@todo more strict requrements for output precision?)
     {{_u8 | _i8, _i8, _any, _any},                       pt(bypass(), bypass(), bypass(), use<3>())},
     // compresses int weights (@todo more strict requrements for output precision?)
-    {{_f32 | _bf16, _u8 | _nf4 | _u4 | _i4, _any, _any}, pt(bypass(), bypass(), use<0>(), use<0>())},
+    {{_bf16, _u8 | _nf4 | _u4 | _i4, _any, _any},        pt(bypass(), bypass(), use<0>(), use<0>()),
+     Require<dnnl::impl::cpu::x64::avx512_core_bf16>()}, // Ticket 122347
```

Comment:
nit, while reading the code this is not so obsious which configuration is really limited. Maybe place `Require` on the same line?

Reply:
Agree, but unfortunately it does not pass the 160 characters limit.
In the future this map will be placed in a separate dnnl specific file as it was discussed. where we can start to use "using namespace"
to make it a bit more compact